targets/thingsplus-rp2040.ld

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
See comment above about not needing this file.

Reply:
see above : flash section is much larger